### PR TITLE
Stackstorm Open-Nebula OpenNebula host counting

### PR DESCRIPTION
**Pull request by ccamp2-7**

### CODE REVIEW

**Comment by jschoewe** [comment]
Can you add your cluster_host_count tests to this branch?

File: actions/cluster_host_count.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+            cluster_value = {}
+            if isinstance(clst.HOSTS.ID, list):
+                cluster_value["host_list"] = clst.HOSTS.ID
+                cluster_value["host_count"] = len(clst.HOSTS.ID)
+            else:
+                cluster_value["host_list"] = [clst.HOSTS.ID]
+                cluster_value["host_count"] = 1
+            return_value[clst.NAME] = cluster_value
+
+        return return_value
```
Comment (jschoewe) [inline actions/cluster_host_count.py:38]: Also need a new line in both of these files

Comment (ccamp2-7) [reply actions/cluster_host_count.py:38]: wdym need a new line?

**Comment by jschoewe** [comment]
LGTM

**Review by bishopbm1 — APPROVED** [review]

**Review by bishopbm1 — APPROVED** [review]

**Pull request merged**